tests/common/plugins/conditional_mark/tests_mark_conditions.yaml

```diff
@@ -2785,6 +2852,13 @@ lldp/test_lldp.py::test_lldp_neighbor_post_orchagent_reboot:
 #######################################
 #####           macsec            #####
 #######################################
+
+macsec:
+  skip:
+    reason: "Test is skipped for Mellanox SN5640 platform and for  Arista-7060X6-64PE-B-* TH5 SKUs"
+    conditions:
+      - "hwsku in ['Arista-7060X6-64PE-B-C448O16','Arista-7060X6-64PE-B-C512S2','Arista-7050CX3-32S-C32','Arista-7050CX3-32S-D48C8','Arista-7260CX3-D108C8','Arista-7260CX3-C64','Arista-7050CX3-32C-C32','Mellanox-SN4700-V64','Cisco-8101-O8C48','Cisco-8101-V64','Cisco-8101C01-C32','Mellanox-SN5640-C512S2','Mellanox-SN5640-C448O16']"
```

Comment:
`conditions_logical_operator: "OR"`

- "hwsku in ['Arista-7060X6-64PE-B-C448O16','Arista-7060X6-64PE-B-C512S2','Mellanox-SN4700-V64','Mellanox-SN5640-C512S2','Mellanox-SN5640-C448O16']"
- "'dualtor' in topo_name"